recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ package:
   version: {{ version }}
 
 source:
-  - url: https://sourceware.org/pub/gcc/releases/gcc-{{ version }}/gcc-{{ version }}.tar.gz
+  - url: https://ftp.gnu.org/gnu/gcc/gcc-{{ version }}/gcc-{{ version }}.tar.gz
```

Comment:
FWIW, 12.4 wasn't available on the ftp server shortly after release, and the [announcement](https://gcc.gnu.org/pipermail/gcc/2024-June/244173.html) made it seem like that won't happen (also my [understanding](https://sourceware.org/sourceware-security-vision.html) of sourceware was that it's much closer to GCC than third-party). Just for context why I changed that.

Reply:
Ah, thanks. I didn't see the FSF affiliation.